hydra-plutus/src/Hydra/Contract/Commit.hs

```diff
@@ -8,20 +8,17 @@ module Hydra.Contract.Commit where
 import Ledger hiding (validatorHash)
 import PlutusTx.Prelude
 
+import Hydra.Contract.HeadState (State (..))
 import Hydra.Data.Party (Party)
-import Ledger.Typed.Scripts (TypedValidator, ValidatorTypes (..))
+import Ledger.Typed.Scripts (TypedValidator, ValidatorType, ValidatorTypes (..))
 import qualified Ledger.Typed.Scripts as Scripts
+import Plutus.V1.Ledger.Api (Credential (ScriptCredential))
+import PlutusTx (CompiledCode)
 import qualified PlutusTx
-import PlutusTx.IsData.Class (ToData (..))
+import PlutusTx.IsData.Class (FromData (fromBuiltinData), ToData (..))
 
 data Commit
 
-data CommitRedeemer
-  = Abort
-  | Collect
```

Comment:
Why did you remove the `CommitRedeemer`?

Reply:
It's not used: The commit script checks the head state is `Initial` and the only transitions out of `Initial` are `CollectCom` or `Abort`.